docs/INSTALLATION.md

```diff
@@ -152,6 +152,7 @@ Daarnaast zijn er bronnen die binnen KISS doorzocht moeten worden.
 | `ENTERPRISE_SEARCH_ENGINE`              | De naam van de `meta-engine` engine die KISS gebruikt. Dit moet zijn: `KISS-engine`  <details> <summary>Meer informatie </summary> De KISS-Elastic-Sync maakt deze engine aan, als deze nog niet bestaat.  </details>   |
 | `KLANTINTERACTIES_BASE_URL`                      | URL van de Klantinteractie API van het gebruikte klantinteractieregister (bijvoorbeeld Open Klant 2) <details> <summary>Meer informatie </summary>Bijvoorbeeld `https://klantinteractieregister.mijngemeente.nl/klantinteracties` </details>  |
 | `KLANTINTERACTIES_TOKEN`                     | Token voor de Klantinteractie API van het gebruikte klantinteractieregister                                                                   |
+| `USE_KLANTINTERACTIES` | Deze variabele bepaalt of er gebruik wordt gemaakt van OpenKlant 2.0 (klantinteractieregister).<details> <summary>Meer informatie </summary>Als deze variabele op `true` staat, wordt OpenKlant 2.0 gebruikt. Standaard is deze waarde `false`.</details>|
 | `CONTACTMOMENTEN_BASE_URL`              | URL van de Contactmomenten API  <details> <summary>Meer informatie </summary>Bijvoorbeeld: `https://contactmomentenregister.mijngemeente.nl` </details>                  |

```

Comment:
Is het verstandig om hier ook te vermelden dat bij de waarde 'false' de oude Contactmomenten API wordt gebruikt? En dat deze aansluit op de e-Suite?

Reply:
Ik ga dat nu nog even toevoegen.